drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -240,7 +240,17 @@ def load_definition(form_definition:, identifier:, role:, title: nil)
       # Apply any client-specific patches
       apply_all_patches!(form_definition, identifier: identifier)
       # Validate final definition
-      validate_definition(form_definition, role)
+      begin
+        validate_definition(form_definition, role)
+      rescue StandardError => e
```

Comment:
This is fine as-is. One thing that maybe worth doing rescuing a custom error rather than StandardError; that's a little more constrained.  Essentially:

```suggestion
      rescue JsonFormException => e
```

would also need to add 
```
module HmisUtil
  class JsonForms
    JsonFormException = Class.new(StandardError)
    private_constant :JsonFormException
```
and then change validate_definition
```diff
-      on_error = ->(err) { block_given? ? yield(err) : raise(err) }
+      on_error = ->(err) { block_given? ? yield(err) : raise(JsonFormException, err) }
```

Reply:
Thanks for the suggestion, added! I didn't know about `private_constant` thats useful.